tests/main/interfaces-network-control/task.yaml

```
@@ -54,6 +54,9 @@ execute: |
     echo "Then the snap command can query network status information"
     network-control-consumer.cmd ss -lnt | MATCH "LISTEN.*:$PORT"
 
+    echo "And DNS information"
+    network-control-consumer.cmd resolvectl | MATCH "DNS Server"
```

This fails on centos 7, 8, debian 11, arch, and amazon linux. The images apparently do not have systemd-resolved installed.
```suggestion
    case "$SPREAD_SYSTEM" in
    centos-*|debian-*|arch-linux-*|amazon-linux-*)
        # echo no systemd-resolved in those images
        ;;
    *)
        network-control-consumer.cmd resolvectl | MATCH "DNS Server"
        ;;
    esac
```

---

Thanks!